StripeFinancialConnections/StripeFinancialConnections/Source/FinancialConnectionsSheet.swift

```diff
@@ -211,8 +211,9 @@ final public class FinancialConnectionsSheet {
             }
         }
 
+        let financialConnectionsApiClient = FinancialConnectionsAPIClient(apiClient: apiClient)
```

Comment:
oh so this is OK to do (you don't need code in `didSet`) _because_ this code only gets executed when we present the sheet (not when we initialize the sheet), so the presence of `apiClient` being a `public` key at this point (and its chance of being updated) doesn't matter 👍